src/include/main/client_config.h

```diff
@@ -13,7 +13,7 @@ struct ClientConfigDefault {
     static constexpr uint64_t TIMEOUT_IN_MS = 0;
     static constexpr uint32_t VAR_LENGTH_MAX_DEPTH = 30;
     static constexpr bool ENABLE_SEMI_MASK = true;
-    static constexpr bool ENABLE_ZONE_MAP = false;
+    static constexpr bool ENABLE_ZONE_MAP = true;
```

Comment:
Should probably keep this disabled by default until this also works for rel tables